hoomd/typeconverter.py

```diff
@@ -1,82 +1,77 @@
 from numpy import array, ndarray
```

Comment:
Add docstrings so that developers understand how these functions and classes should be used.

Reply:
Good idea done.